package.json

```diff
@@ -36,9 +36,6 @@
     "zeplin:tokens": "node scripts/zeplin-styleguide.js"
   },
   "license": "Apache-2.0",
-  "dependencies": {
-    "@csstools/normalize.css": "^12.0.0"
```

Comment:
if we remove the dependency in total, how would we get to know any updates? And how would the update process work in general?